plugins/hydra_ray_launcher/examples/upload_download/conf/hydra/launcher/custom_ray_aws.yaml

```diff
@@ -9,12 +9,11 @@ sync_up:
   # we leave target_dir to be null
   # as a result the files will be synced to a temp dir on remote cluster.
   # the temp dir will be cleaned up after the jobs are done.
-  # recommend to leave target_dir to be null if you are syncing code/artifacts to remote cluster so you don't need
+  # recommend to leave target_dir to be null if you are syncing code/artifacts
+  # to remote cluster so you don't need
   # configure $PYTHONPATH on remote cluster
-  include: ['model', '*.py']
   # No need to sync up config files.
-  exclude: ['*']
+  exclude: ['conf', 'multirun', "**/__pycache__"]
```

Comment:
I'd prefer not to hardcode `multirun` here, since people can override output dir, and `multirun` won't necesserily be their output dir always.